lib/gateways/pull_request.rb

```diff
@@ -14,7 +14,7 @@ def execute
     end
 
     def approved_pull_requests
-      query = "is:pr user:alphagov state:open author:app/dependabot author:app/dependabot-preview review:approved"
+      query = "is:pr user:alphagov state:open author:app/dependabot review:approved"
```

Comment:
Nice improvement 👍 
Nitpick: could you link to some evidence in the commit message?

Reply:
I added a link to a PR that has a bit more info